### DISCUSSION TITLE
I have not received my education benefits after I was approved in Nov 23

### DISCUSSION THREAD

**Discussion by cy-Yin** [community/community#145640] [GitHub Education] [answered]
### Hi everyone,

![Screenshot_20241126_183606](https://github.com/user-attachments/assets/ac8ff4ca-24cb-413f-824a-9772a2c29749)

I hope someone can fix this problem for me. I have been approved for GitHub education program on 23th November as you can see in the attached screenshot, but I have not received the benefits yet. No email as well. Is there anything wrong?

**Comment by davevad93** [comment, marked as answer]
Hi @cy-Yin ,

Don't worry it's normal, when the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @cy-Yin was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by cy-Yin** [reply]
Wow. Thanks a lot. It seems that I am allowed to use GitHub education now.
